src/frontend/src/pages/Index/Scan.tsx

```diff
@@ -177,11 +174,7 @@ export default function Scan() {
           <Trans>Actions ... </Trans>
         </Text>
         <Group>
-          <ActionIcon
-            onClick={notYetImplemented}
-            title={t`Count`}
-            variant='default'
-          >
+          <ActionIcon onClick={() => {}} title={t`Count`} variant='default'>
```

Comment:
Comment: won't we forget about these if they are not marked anymore?

Reply:
Good point, have reverted